pwa/src/templates/templateParts/footer/FooterTemplate.module.css

```diff
@@ -14,6 +14,11 @@
 
 .imagesContainer {
   display: flex;
+  cursor: pointer;
```

Comment:
Because you've set this rule to the entire `imagesContainer`, the cursor becomes a pointer anywhere when hovering anywhere over the container. This means that the user gets the visual feedback of a changing cursor when there's nothing yet clickable.

To fix this, we can set it to only the logo's. Also, we can only set it on the `:hover` event, like so:

```
.imagesContainer > *:hover {
  cursor: pointer;
}
```